src/ProgressOnderwijsUtils/MetaObject/MetaObjectBulkCopy.cs

```diff
@@ -22,7 +22,7 @@ public static class MetaObjectBulkCopy
         /// <param name="tableName">The name of the table to import into; must be a valid sql identifier (i.e. you must escape special characters if any).</param>
         public static void BulkCopyToSqlServer<T>(this IEnumerable<T> metaObjects, SqlCommandCreationContext sqlconn, string tableName) where T : IMetaObject, IPropertiesAreUsedImplicitly
         {
-            using (var bulkCopy = new SqlBulkCopy(sqlconn.Connection, SqlBulkCopyOptions.CheckConstraints | SqlBulkCopyOptions.UseInternalTransaction, null)) {

```

Comment:
Je hebt alle gebruikers hiervan iig vlug geskimmed?

Reply:
Connections met P3W worden wel vaak zonder transacties gebruikt. Willen we dat veranderen?

Reply:
Lijkt me wel iets om te overwegen; maar de meeste zijn vermoedelijk readonly, muv de nachtelijk sync, en die maakt toch gebruik van een last-minute swap, dus alles voor die swap is effectief transactioneel daardoor, ook zonder sql transactions.